### PR TITLE
Remove remark, update linters and update lint-staged commands

### PR DESCRIPTION
**Pull request by dhogan8**

### CODE REVIEW

File: content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx [hunk 1]
```diff
@@ -10,22 +10,16 @@ import responseJson from '../_examples/response';
...
-      * `is_public_proxy`
-      * `is_residential_proxy`
-      * `is_tor_exit_node`
-  {/* <!--lint enable list-item-bullet-indent --> */}
+1. `risk` has been added directly to the `ip_address` object
+1. `local_time` has been added to the `location` sub-object
+1. The `maxmind` object is not present. See below for descriptions.
+1. minFraud Insights and Factors return the following anonymous IP outputs: _
+   `is_anonymous` _ `is_anonymous_vpn` _ `is_hosting_provider` _
+   `is_public_proxy` _ `is_residential_proxy` _ `is_tor_exit_node`
```
Comment (mm-jpoole) [inline content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx:18]: This one is kinda weird. It was an unordered (sub)list, but now it is just rendered on a single line separated with underscores.

I think it should probably be changed back to a list.

File: content/minfraud/release-notes/2021.mdx [hunk 2]
```diff
@@ -28,74 +29,83 @@ Private Relay works and delivers the best possible user experience for your
...
-    - `/shipping_address/distance_to_ip_location`
-    - `/shipping_address/distance_to_billing_address`
-    - `/billing_address/distance_to_ip_location`
+- We added `BILLING_REGION_NOT_FOUND` and `SHIPPING_REGION_NOT_FOUND` codes
+- We updated the warning explanations for certain warning codes to include that
+  distance calculations (outlined below) may be impacted when certain location
+  information is missing or cannot be found -
+  `/shipping_address/distance_to_ip_location` -
+  `/shipping_address/distance_to_billing_address` -
+  `/billing_address/distance_to_ip_location`
```
Comment (mm-jpoole) [inline content/minfraud/release-notes/2021.mdx:104]: Similar issue here: used to be a list, but now it is not. Should probably be changed back to a list.

File: content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx (hunk 1 above)
Comment (oschwald) [reply content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx:18]: I am guessing adding a empty line before the list would prevent it from doing this. Some Markdown parsers require this.

**Review by mm-jpoole — CHANGES_REQUESTED** [review]
Just the two lists that got mangled

File: content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx [hunk 3]
```diff
@@ -11,11 +11,16 @@ import responseJson from '../_examples/response';
...
+2. `local_time` has been added to the `location` sub-object
+3. The `maxmind` object is not present. See below for descriptions.
+4. minFraud Insights and Factors return the following anonymous IP outputs:
+
+- `is_anonymous`
+- `is_anonymous_vpn`
+- `is_hosting_provider`
+- `is_public_proxy`
+- `is_residential_proxy`
+- `is_tor_exit_node`
```
Comment (dhogan8) [inline content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx:23]: @oschwald @mm-jpoole 

Screen Shot 2022-09-30 at 6 48 44 PM

I tried applying the following, but it reverts to the above and as a result doesn't look like what it should here: https://dev.maxmind.com/minfraud/api-documentation/responses?lang=en#schema--response--ip-address:

```
diff --git a/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx b/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx
index f42fd4f..3bce3e4 100644
--- a/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx
+++ b/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx
@@ -15,12 +15,12 @@ import responseJson from '../_examples/response';
 3. The `maxmind` object is not present. See below for descriptions.
 4. minFraud Insights and Factors return the following anonymous IP outputs:

-- `is_anonymous`
-- `is_anonymous_vpn`
-- `is_hosting_provider`
-- `is_public_proxy`
-- `is_residential_proxy`
-- `is_tor_exit_node`
+  - `is_anonymous`
+  - `is_anonymous_vpn`
+  - `is_hosting_provider`
+  - `is_public_proxy`
+  - `is_residential_proxy`
+  - `is_tor_exit_node`
```

Comment (dhogan8) [reply content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx]: Sorry, never mind. 2 indents did the trick in https://github.com/maxmind/dev-site/pull/826/commits/a4455594a887274123027cd3151fe30c0aec4e97.

**Review by mm-jpoole — APPROVED** [review]
Looking good!

**Pull request merged**